contracts/core/VaultRouter.sol

```diff
@@ -421,12 +420,11 @@ contract VaultRouter is
     IERC4626 vault;
     uint256 shares;
     address receiver;
-    address owner;
+    address owner = msg.sender;
```

Comment:
```suggestion
    address owner;
```
It might be better to allow the owner to be specified in the calldata, is more flexibility for all transactions.